ci/test/mkpipeline.py

```diff
@@ -139,6 +139,22 @@ def inputs(self) -> Set[str]:
         return inputs
 
 
+def prioritize_pipeline(pipeline: Any) -> None:
+    """Prioritize builds against main or release branches"""
+
+    branch = os.environ["BUILDKITE_BRANCH"]
+    priority = None
+
+    if branch.startswith("v"):
```

Comment:
Shall we instead use a regex here to expect at least one number after `v` to avoid picking up all branches starting with `v`? (optional)